### PR TITLE
Improve Animation SEXP usability

### PR DESCRIPTION
**Pull request by BMagnu**
This adds better description to the new SEXPs, adds proper error checking for the entered animation triggers, and shows the FREDer the available triggers / names.

### CODE REVIEW

File: code/parse/sexp.cpp [hunk 1]
```diff
@@ -35835,7 +35933,8 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"Takes 3 or more arguments...\r\n"
 		"\t1: The ship to trigger the animation on.\r\n"
 		"\t2: The trigger type of the animation.\r\n"
-		"\t3: The triggered-by value of the animation. Depends on trigger type. Refer to the wiki on *-anim.tbm's.\r\n"
+		"\t3: The triggered-by value of the animation. Must be the same as in the table for the animation. Leave blank if not specified in the table.\r\n"
```
Comment (Goober5000) [inline code/parse/sexp.cpp:35936]: Does the syntax checking correctly handle the blank string?

**Review by Goober5000 — COMMENTED** [review]
Looks good, but one question.

**Comment by BMagnu** [comment]
In this case yes, cause if they don't have that specified, they'll have a default subtype. This causes them to get validated by trigger string which is then empty, thus allowing empty animation if such an animation exists.On 30 Jan 2022 1:13 am, Goober5000 ***@***.***> wrote:
@Goober5000 commented on this pull request.

Looks good, but one question.

In code/parse/sexp.cpp:
> @@ -35835,7 +35933,8 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"Takes 3 or more arguments...\r\n"
 		"\t1: The ship to trigger the animation on.\r\n"
 		"\t2: The trigger type of the animation.\r\n"
-		"\t3: The triggered-by value of the animation. Depends on trigger type. Refer to the wiki on *-anim.tbm's.\r\n"
+		"\t3: The triggered-by value of the animation. Must be the same as in the table for the animation. Leave blank if not specified in the table.\r\n"

Does the syntax checking correctly handle the blank string?

**Review by Goober5000 — APPROVED** [review]
Ok sounds good.

**Pull request merged**